static/search.js

```diff
@@ -124,29 +130,11 @@ var Search = Control.extend({
 	// ---- SETUP / TEARDOWN ---- //
 
 
-//  ---- LOCAL STORAGE ---- //
-	getLocalStorageItem: function(key){
-		if(!window.localStorage){
-			return null;
-		}
-
-		var storageItem = localStorage.getItem(key);
-
-		if(storageItem){
-			return JSON.parse(storageItem);	
-		}
-
-		return null;
+	//  ---- LOCAL STORAGE ---- //
+	// Init with noop functions, then overwrite if localStorage is supported
```

Comment:
Instead of noop functions, it would be handy if just an object was used. I would keep all of that logic in the two functions and move their definitions back to this section of the code.